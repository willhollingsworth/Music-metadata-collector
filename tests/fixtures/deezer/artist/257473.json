{
  "id": 257473,
  "name": "Paul Lomax",
  "link": "https://www.deezer.com/artist/257473",
  "share": "https://www.deezer.com/artist/257473?utm_source=deezer&utm_content=artist-257473&utm_term=0_1751622005&utm_medium=web",
  "picture": "https://api.deezer.com/artist/257473/image",
  "picture_small": "https://cdn-images.dzcdn.net/images/artist//56x56-000000-80-0-0.jpg",
  "picture_medium": "https://cdn-images.dzcdn.net/images/artist//250x250-000000-80-0-0.jpg",
  "picture_big": "https://cdn-images.dzcdn.net/images/artist//500x500-000000-80-0-0.jpg",
  "picture_xl": "https://cdn-images.dzcdn.net/images/artist//1000x1000-000000-80-0-0.jpg",
  "nb_album": 10,
  "nb_fan": 217,
  "radio": true,
  "tracklist": "https://api.deezer.com/artist/257473/top?limit=50",
  "type": "artist"
}