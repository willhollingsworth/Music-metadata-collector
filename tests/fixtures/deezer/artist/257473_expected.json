{
  "artist_name": "Paul Lomax",
  "artist_id": 257473,
  "track_count": 10,
  "fans_count": 217,
  "link": "https://www.deezer.com/artist/257473"
}